mysql-test/suite/rpl/t/rpl_percona_userstat.test

```diff
@@ -0,0 +1,60 @@
+--echo #
+--echo # Bug 995624 RBR events are not reflected in userstat's
+--echo # Rows_updated
+--echo #
+
+--source include/master-slave.inc
+
+connection slave;
+set @slave_userstat_saved= @@global.userstat;
+set global userstat=ON;
+--echo Rows_updated on slave - starting situation
+select ROWS_UPDATED from information_schema.client_statistics;
```

Comment:
Continuing discussion on the closed 5.6 PR, I'd replace the starting / ending situation records with include/assert.inc check. Let's discuss on Slack if needed